.ocamlformat

```diff
@@ -0,0 +1,3 @@
+version=0.12
+profile=conventional
+margin=100
```

Comment:
why 100?

Reply:
Personal preference, I'm happy to remove it if anybody objects!

Reply:
I personally like to keep the default 80 col width in my editor, but I don't know how many people do that as well.

Reply:
Let's stick to the default then!